Makefile

```diff
@@ -67,7 +67,7 @@ vet: ## Run go vet against code.
 
 .PHONY: test
 test: manifests generate fmt vet envtest ## Run tests.
-	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) --bin-dir $(LOCALBIN) -p path)" go test $$(go list ./... | grep -v /e2e | grep -v /test/utils) -coverprofile cover.out
+	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) --bin-dir $(LOCALBIN) -p path)" go test $$(go list ./... | grep -v /e2e | grep -v /test/utils) -coverprofile cover.out ./...
```

Comment:
What is the purpose of this change, isn't the go list command supposed to get all the modules to test (minus the excluded ones)?

Reply:
This I was testing when i encoundered some errors. I will revert back

Reply:
done